sdk-actors/src/main/java/io/dapr/actors/ActorUtils.java

```diff
@@ -49,4 +56,33 @@ public static String findActorTypeName(Class<?> actorClass) {
     ActorType actorTypeAnnotation = node.getAnnotation(ActorType.class);
     return actorTypeAnnotation != null ? actorTypeAnnotation.name() : actorClass.getSimpleName();
   }
+
+  /**
+   * Retrieves sdk version from resources.
+   *
+   * @return String version of sdk.
+   */
+  public static String getVersion() {
+
+    if (version != null) {
+      return version;
+    }
+
+    InputStream in = new ByteArrayInputStream(new byte[0]);
+    try {
+      Properties properties = new Properties();
+      in = ActorUtils.class.getResourceAsStream("/app.properties");

```

Comment:
Make it a try with resources block so that input stream is properly closed even on exception

Reply:
Oh cool, wasn't aware of this feature.